datasets/pn_summary/README.md

```diff
@@ -0,0 +1,183 @@
+| 2 | 00fa692a17 ...  | سبد محصولات پتروشیمی متنوع می‌شود                 | به گزارش شانا به نقل از شرکت ملی صنایع پتروشیمی، علی‌اصغر گودرزی‌فراهانی با اشاره به اینکه همه طرح‌های در حال اجرای صنعت پتروشیمی براساس پیشرفت فیزیکی و پیش‌بینی زمان راه‌اندازی در قالب طرح‌های جهش دوم و سوم تقسیم‌بندی شده‌اند، اظهار کرد: انتظار داریم که طرح‌های جهش دوم صنعت پتروشیمی که پیشرفت‌های (...)                                                                               | سرپرست مدیریت برنامه‌ریزی و توسعه شرکت ملی صنایع پتروشیمی گفت: تنوع محصولات پتروشیمی ایران با بهره‌برداری از طرح‌های جهش دوم و سوم صنعت پتروشیمی افزایش می‌یابد.                                           | Oil-Energy | پتروشیمی   | Shana   | https://www.shana.ir ...  |
+
+
+### Data Fields
+
+- `id`: id of the news.
+- `title`: the title of the news.
+- `article`: the article of the news.
+- `summary`: the summary of the news.
+- `category`: the category of news in English.
+- `categories`: the category and sub-category of the news in Persian.
+- `network`: the name of the news agency.
+- `link`: the link of the news.
+
+The category in English includes 18 different article categories from economy to tourism. 
+
+```bash
+Economy, Roads-Urban, Banking-Insurance, Agriculture, International, Oil-Energy, Industry, Transportation, Science-Technology, Local, Sports, Politics, Art-Culture, Society, Health, Research, Education-University, Tourism
+```
+
+### Data Splits
+
+Training (82,022 records, 8 features), validation (5,592 records, 8 features), and test split (5,593 records and 8 features).
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+The dataset comprises numerous articles of various categories that have been crawled from six news agency websites (Tahlilbazaar, Imna, Shana, Mehr, Irna, and Khabaronline).
+
+### Annotations
+
+#### Annotation process
+
+Each record (article) includes the long original text as well as a human-generated summary. The total number of cleaned articles is 93,207 (from 200,000 crawled articles).
+
+#### Who are the annotators?
+
+The dataset was organized by [Mehrdad Farahani](https://github.com/m3hrdadfi), [Mohammad Gharachorloo](https://github.com/baarsaam) and [Mohammad Manthouri](https://github.com/mmanthouri) for this paper [Leveraging ParsBERT and Pretrained mT5 for Persian Abstractive Text Summarization](https://arxiv.org/abs/2012.11204)
+
+### Personal and Sensitive Information
+
+[Needs More Information]
```

Comment:
```suggestion
[More Information Needed]
```
We usually spell it `[More Information Needed]` instead of `[Needs More Information]` in all the other datasets

Reply:
I didn't know, used the suggested generator, but I think you should probably change the template on the online [card generator](https://huggingface.co/datasets/card-creator/) as well.

Reply:
Indeed ^^
well it's fine then